apps/sms/js/thread_list_ui.js

```diff
@@ -646,7 +648,7 @@ var ThreadListUI = {
     Settings.setReadAheadThreadRetrieval(this.FIRST_PANEL_THREAD_COUNT);
   },
 
-  renderThreads: function thlui_renderThreads(firstViewDoneCb, allDoneCb) {
+  renderThreads: function thlui_renderThreads(firstViewDoneCb) {
```

Comment:
Yeah, it feels we need smth like this, it will look better than callback + promise, I haven't seen any draft spec about progress events on promises, maybe we can shim it somehow and use for this case and for our cursors - just idea need to find any pointers first